src/features/metadata-instance-editor/Instance.js

```diff
@@ -52,8 +52,9 @@ type Props = {
     hasError: boolean,
     id: string,
     intl: Object,
-    isCascadingPolicyApplicable?: boolean,
+    isCascadingPolicyApplicable: boolean,
```

Comment:
I this intentional? `metadata.js` is still using `isCascadingPolicyApplicable?: boolean,`